docs/reference/settings/security-settings.asciidoc

```diff
@@ -1501,6 +1501,8 @@ This setting cannot be used with `ssl.truststore.password`.
 // tag::saml-ssl-verification-mode-tag[]
 `ssl.verification_mode` {ess-icon}::
 (<<static-cluster-setting,Static>>)
+Indicates the type of verification when using `saml` to protect against man
+in the middle attacks and certificate forgery.
```

Comment:
IIUC, this is adopted from the same setting of LDAP/AD realms. I don't think we need it here. LDAP and AD can both use either `ldap` or `ldaps` protocol with `ldaps` being the ldap over SSL. So it makes sense to say

> .. when using `ldaps` to protect against man in the middle attacks ...

Replacing `ldaps` with `saml` does not read right because SAML is not "something over SSL". In fact, SAML does not realy care about SSL in ES's use case because it makes no back-channel HTTP call. The only thing relevant is where to retrieval the SAML metadata, which could be from a HTTP URL. This setting, in fact this whole sub-section about SSL, is to control HTTP call for retrieving the metadata. This is stated in beginning of this sub-section (Line 1401). So it is not necessary to call it out here.

Reply:
Thanks for that explanation @ywangd -- I changed this instances (and the other related ones) to just read, "Controls the verification of certificates."